src/routes/blog/index.svelte

```diff
@@ -31,51 +31,46 @@
 </script>
 
 <svelte:head>
-	<title>Files - Blog</title>
-	<meta content="Files - Blog" name="og:title"/>
-	<meta content="Files - Blog" name="twitter:title"/>
-
-	<meta content="/branding/banner-blog-light.png" name="og:image"/>
-	<meta content="https://{$page.host}/branding/banner-blog-light.png" name="twitter:image"/>
+	<Metadata title="Files • Blog" image="blog" />
```

Comment:
Same here, was there a specific reason to use the bullet point instead of hyphen?